 --- T2-COPYRIGHT-NOTE-BEGIN ---
 This copyright note is auto-generated by ./scripts/Create-CopyPatch.
 
 T2 SDE: package/.../linux24/06-devfs-default.patch
 Copyright (C) 2004 - 2005 The T2 SDE Project
 Copyright (C) 1998 - 2003 ROCK Linux Project
 
 More information can be found in the files COPYING and README.
 
 This patch file is dual-licensed. It is available under the license the
 patched project is licensed under, as long as it is an OpenSource license
 as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
 of the GNU General Public License as published by the Free Software
 Foundation; either version 2 of the License, or (at your option) any later
 version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -rduN ./arch/alpha/defconfig ./arch/alpha/defconfig
--- ./arch/alpha/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/alpha/defconfig	2004-04-20 00:03:27.000000000 +0200
@@ -567,8 +567,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/arm/defconfig ./arch/arm/defconfig
--- ./arch/arm/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/arm/defconfig	2004-04-20 00:13:05.000000000 +0200
@@ -422,8 +422,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/cris/defconfig ./arch/cris/defconfig
--- ./arch/cris/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/cris/defconfig	2004-04-20 00:18:07.000000000 +0200
@@ -457,8 +457,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 # CONFIG_DEVPTS_FS is not set
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/i386/defconfig ./arch/i386/defconfig
--- ./arch/i386/defconfig	2004-04-14 15:05:25.000000000 +0200
+++ ./arch/i386/defconfig	2004-04-20 00:01:37.000000000 +0200
@@ -711,8 +711,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/ia64/defconfig ./arch/ia64/defconfig
--- ./arch/ia64/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/ia64/defconfig	2004-04-20 00:15:28.000000000 +0200
@@ -721,8 +721,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/m68k/defconfig ./arch/m68k/defconfig
--- ./arch/m68k/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/m68k/defconfig	2004-04-20 00:07:43.000000000 +0200
@@ -229,8 +229,8 @@
 # CONFIG_NTFS_FS is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/mips/defconfig ./arch/mips/defconfig
--- ./arch/mips/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/mips/defconfig	2004-04-20 00:05:33.000000000 +0200
@@ -577,8 +577,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/mips64/defconfig ./arch/mips64/defconfig
--- ./arch/mips64/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/mips64/defconfig	2004-04-20 00:16:20.000000000 +0200
@@ -533,8 +533,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/parisc/defconfig ./arch/parisc/defconfig
--- ./arch/parisc/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/parisc/defconfig	2004-04-20 00:17:32.000000000 +0200
@@ -617,8 +617,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/ppc/defconfig ./arch/ppc/defconfig
--- ./arch/ppc/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/ppc/defconfig	2004-04-20 00:06:02.000000000 +0200
@@ -789,7 +789,7 @@
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
 CONFIG_DEVFS_FS=y
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/sh/defconfig ./arch/sh/defconfig
--- ./arch/sh/defconfig	2004-02-18 14:36:30.000000000 +0100
+++ ./arch/sh/defconfig	2004-04-20 00:14:05.000000000 +0200
@@ -167,8 +167,8 @@
 # CONFIG_NTFS_RW is not set
 # CONFIG_HPFS_FS is not set
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 # CONFIG_DEVPTS_FS is not set
 # CONFIG_QNX4FS_FS is not set
diff -rduN ./arch/sparc/defconfig ./arch/sparc/defconfig
--- ./arch/sparc/defconfig	2004-04-14 15:05:27.000000000 +0200
+++ ./arch/sparc/defconfig	2004-04-20 00:04:42.000000000 +0200
@@ -329,8 +329,8 @@
 # CONFIG_NTFS_RW is not set
 CONFIG_HPFS_FS=m
 CONFIG_PROC_FS=y
-# CONFIG_DEVFS_FS is not set
-# CONFIG_DEVFS_MOUNT is not set
+CONFIG_DEVFS_FS=y
+CONFIG_DEVFS_MOUNT=y
 # CONFIG_DEVFS_DEBUG is not set
 CONFIG_DEVPTS_FS=y
 # CONFIG_QNX4FS_FS is not set
